app/models/manageiq/providers/ibm_cloud/inventory/persister/power_virtual_servers.rb

```diff
@@ -50,4 +51,12 @@ def add_advanced_settings
       )
     end
   end
+
+  def add_snapshots
+    add_collection(cloud, :snapshots) do |builder|
+      builder.add_properties(:model_class => ::Snapshot)
+      builder.add_properties(:parent_inventory_collections => %i[vms miq_templates])
+      builder.add_properties(:complete => !targeted?)
+    end
```

Comment:
These options should be the defaults coming from the core inventory collections: https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/infra_manager.rb#L59-L64